ACEHAL/fit.py

```diff
@@ -320,7 +343,15 @@ def do_fit(Psi, Y, B, E0s, solver, n_committee=8, basis_normalization=None, pot_
         # indicate which ones those are.
         if sigma.shape[0] != len(c_norm):
              # only valid for sklearn ARDRegression
-            included_c = solver.lambda_ < solver.threshold_lambda
+
+             #Skelarn ARD
+            if "threshold_lambda" in solver.__dict__:
+                threshold_lambda = solver.threshold_lambda
+                included_c = solver.lambda_ < solver.threshold_lambda
+            #BRM from Noam and Chucks
+            else:
+                included_c = abs(solver.coef_) > 1e-15
```

Comment:
I'm not sure this is the right criterion in BRM's solver.

```included_c = solver.var_c_ > (solver.var_c_min * solver.threshold)```

This seems more reasonable I think, do you agree?

Reply:
I think the 1e-15 is quite arbitrary and didn't work for me, we should look at the variances for the pruning. Can you fix and push? Maybe add something to look for a `BRM` instance?

Reply:
+ agree this whole thing shouldn't be in ACEHAL but in the solver itself.

Reply:
> Agree the variances should be used for pruning but I'm surprised the using 1e-15 didn't work. BRM zeros out the unused coefficients and surely 1e-15 is tight enough to identify this

Reply:
No problem! I don't understand how this could've worked for you and it certainly didn't for me. I spent some time performing a few fits and on my machine this numbers just didn't match...